damus/Views/Images/ProfilePicImageView.swift

```diff
@@ -40,29 +40,36 @@ struct ProfileImageContainerView: View {
     }
 }
 
+enum NavDismissBarContainer {
+    case fullScreenCarousel
+    case profilePicImageView
+}
+
 struct NavDismissBarView: View {
     
     @Environment(\.presentationMode) var presentationMode
-    let showBackgroundCircle: Bool
+    let navDismissBarContainer: NavDismissBarContainer
     
-    init(showBackgroundCircle: Bool = true) {
-        self.showBackgroundCircle = showBackgroundCircle
+    init(navDismissBarContainer: NavDismissBarContainer) {
+        self.navDismissBarContainer = navDismissBarContainer
     }
     
     var body: some View {
         HStack {
             Button(action: {
                 presentationMode.wrappedValue.dismiss()
             }, label: {
-                if showBackgroundCircle {
+                if case .profilePicImageView = navDismissBarContainer {
```

Comment:
if case ... else case is an antipattern if you're covering all cases. use a switch statement instead.

Reply:
@jb55 I have addressed and pushed the change